frontend/src/employee-frontend/components/messages/MessageContext.tsx

```diff
@@ -112,16 +120,32 @@ export const MessageContextProvider = React.memo(
     const [selectedUnit, setSelectedUnit] = useState<SelectOption | undefined>()
     const { loggedIn } = useContext(UserContext)
 
-    const [accounts, setAccounts] = useState<Result<MessageAccount[]>>(
-      Loading.of()
+    const [nestedAccounts, setNestedMessagingAccounts] = useState<
+      Result<NestedMessageAccount[]>
+    >(Loading.of())
+
+    const [unreadCountsByAccount, setUnreadCountsByAccount] = useState<
+      Result<UnreadCountByAccount[]>
+    >(Loading.of())
+
+    const getNestedAccounts = useRestApi(
+      getNestedMessagingAccounts,
+      setNestedMessagingAccounts
     )
+    const loadNestedAccounts = useDebouncedCallback(getNestedAccounts, 100)
```

Comment:
Q: What's the debounce for, exactly?

Reply:
This is just a direct translation from [master](https://github.com/espoon-voltti/evaka/blob/b6cfb5dcfaf412979fade078801677b7bf92725c/frontend/src/employee-frontend/components/messages/MessageContext.tsx#L120) to the updated terminology. I'm a little doubtful if the debounce is really necessary, but I'd probably discuss that as a separate issue since this PR is already does quite a lot of stuff.

Reply:
:heavy_check_mark: fine by me